sopel/config/core_section.py

```diff
@@ -235,34 +264,42 @@ def homedir(self):
                                     'WARNING')
     """The lowest severity of logs to display.
 
-    If not specified, this defaults to WARNING.
+    If not specified, this defaults to ``WARNING``.
     """
 
     modes = ValidatedAttribute('modes', default='B')
     """User modes to be set on connection."""
 
     name = ValidatedAttribute('name', default='Sopel: https://sopel.chat')
-    """The "real name" of your bot for WHOIS responses."""
+    """The "real name" of your bot for ``WHOIS`` responses."""
 
     nick = ValidatedAttribute('nick', Identifier, default=Identifier('Sopel'))
-    """The nickname for the bot"""
+    """The nickname for the bot."""
 
     nick_auth_method = ChoiceAttribute('nick_auth_method', choices=[
         'nickserv', 'authserv', 'Q', 'userserv'])
     """The nick authentication method.
 
     Can be ``nickserv``, ``authserv``, ``Q``, or ``userserv``.
+
+    .. versionadded:: 7.0
     """
 
     nick_auth_password = ValidatedAttribute('nick_auth_password')
-    """The password to use to authenticate your nick."""
+    """The password to use to authenticate your nick.
```

Comment:
```suggestion
    """The password to use to authenticate the bot's nick.
```

"your" is confusing, isn't it?